docs/source/whatsnew/version3_widget_migration.rst

```diff
@@ -0,0 +1,319 @@
+widget persistence API, you can persist your widget instances using
+Javascript. This makes it easy to persist your widgets to your notebook
+document (with a small amount of custom JS). By default, the widgets are
+persisted to your web browsers local storage which makes them reappear
+when your refresh the page.
+
+Smaller Changes
+~~~~~~~~~~~~~~~
+
+-  Latex math is supported in widget ``description``\ s
+   (`#5937 <https://github.com/ipython/ipython/pull/5937>`__).
+-  Widgets can be display more than once within a single container
+   widget (`#5963 <https://github.com/ipython/ipython/pull/5963>`__,
+   `#6990 <https://github.com/ipython/ipython/pull/6990>`__).
+-  ``FloatRangeSlider`` and ``IntRangeSlider`` were added
+   (`#6050 <https://github.com/ipython/ipython/pull/6050>`__).
+-  "Widget" was removed from the ends of all of the widget class names
+   (`#6125 <https://github.com/ipython/ipython/pull/6125>`__).
+-  ``ContainerWidget`` was renamed to ``Box``
+   (`#6125 <https://github.com/ipython/ipython/pull/6125>`__).
+-  ``HBox`` and ``VBox`` widgets were added
+   (`#6125 <https://github.com/ipython/ipython/pull/6125>`__).
+-  ``add\_class`` and ``remove\_class`` were removed in favor of a
+   ``_dom_classes`` list
+   (`#6235 <https://github.com/ipython/ipython/pull/6235>`__).
+-  ``get\_css`` and ``set\_css`` were removed in favor of explicit
+   traits for widget styling
+   (`#6235 <https://github.com/ipython/ipython/pull/6235>`__).
+-  ``jslink`` and ``jsdlink`` were added
+   (`#6454 <https://github.com/ipython/ipython/pull/6454>`__,
+   `#7468 <https://github.com/ipython/ipython/pull/7468>`__).
+-  An ``Output`` widget was added, which allows you to ``print`` and
+   ``display`` within widgets
+   (`#6670 <https://github.com/ipython/ipython/pull/6670>`__).
+-  ``PopupWidget`` was removed
+   (`#7341 <https://github.com/ipython/ipython/pull/7341>`__).
+-  A visual cue was added for widgets with 'dead' comms
+   (`#7227 <https://github.com/ipython/ipython/pull/7227>`__).
+-  A ``SelectMultiple`` widget was added (a ``Select`` widget that
+   allows multiple things to be selected at once)
+   (`#6890 <https://github.com/ipython/ipython/pull/6890>`__).
+-  A class was added to help manage children views
+   (`#6990 <https://github.com/ipython/ipython/pull/6990>`__).
+-  A warning was added that shows on widget import because it's expected
+   that the API will change again by IPython 4.0. This warning can be
+   supressed (`#7107 <https://github.com/ipython/ipython/pull/7107>`__,
+   `#7200 <https://github.com/ipython/ipython/pull/7200>`__,
+   `#7201 <https://github.com/ipython/ipython/pull/7201>`__,
+   `#7204 <https://github.com/ipython/ipython/pull/7204>`__).
+
+Comm and Widget PR Index
+------------------------
+
+| Here is a chronological list of PRs affecting the widget and comm
+  frameworks for IPython 3.0. Note that later PRs may revert changes
+  made in earlier PRs:
+| - Add placeholder attribute to text widgets
```

Comment:
I think if you get rid of the | characters and add a blank line before the first bullet point, it should render as a proper bulleted list.